proposals/java_agents_and_more_layers.md

```diff
@@ -58,7 +58,7 @@ addFile 'path/to/file', '/path/on/image'
 addFile 'another/file', '/another/path/on/image'
 ```
 
-All of the files would be added to a single new layer.
+*Directories will be handled using [rsync](https://linux.die.net/man/1/rsync) source-path semantics, where, for example, `addFile 'path/to/directory', '/path/on/image'` would put all the files in `path/to/directory` into the directory `/path/on/image/directory`, but `addFile 'path/to/directory', '/path/on/image'` (trailing slash) would put all the files into the directory `/path/on/image`.* 
```

Comment:
Second example is missing the trailing slash!

Reply:
Oops, will add that.